scripts/test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+process.env.NODE_ENV = 'test';
```

Comment:
Would you mind documenting the behavior of NODE_ENV somewhere like in the README? I know you didn't start using it, but it seems to be undocumented at the moment.